src/NosCore.Parser/Parsers/NpcMonsterParser.cs

```diff
@@ -0,0 +1,495 @@
+                2518 => 911,
+                2519 => 845,
+                2520 => 3682,
+                2521 => 401,
+                2522 => 471,
+                2523 => 328,
+                2524 => 12718,
+                2525 => 412,
+                2526 => 11157,
+                2527 => 18057,
+                2530 => 28756,
+                2559 => 1308,
+                2560 => 1234,
+                2561 => 1168,
+                2562 => 959,
+                2563 => 947,
+                2564 => 952,
+                2566 => 1097,
+                2567 => 1096,
+                2568 => 4340,
+                2569 => 3534,
+                2570 => 4343,
+                2571 => 2205,
+                2572 => 5632,
+                2573 => 3756,
+                _ => 0
+            });
+        }
+
+        private short Level(Dictionary<string, string[][]> chunk)
+        {
+            return Convert.ToInt16(chunk["LEVEL"][0][2]);
+        }
+
+        private long LoadUnknownData(Dictionary<string, string[][]> chunk)
+        {
+            return Convert.ToInt64(chunk["ETC"][0][2]);
+        }
+        private bool ShouldLoadPetinfo(Dictionary<string, string[][]> chunk)
+        {
+            var unknownData = LoadUnknownData(chunk);
+            return ((unknownData != -2147481593) && (unknownData != -2147481599) && (unknownData != -1610610681));
+        }
+
+        public void InsertNpcMonsters(string folder)
+        {
+            var actionList = new Dictionary<string, Func<Dictionary<string, string[][]>, object>>
+            {
+                {nameof(NpcMonsterDto.NpcMonsterVNum), chunk => Convert.ToInt16(chunk["VNUM"][0][2])},
+                {nameof(NpcMonsterDto.NameI18NKey), chunk => chunk["NAME"][0][2]},
+                {nameof(NpcMonsterDto.Level), chunk => Level(chunk)},
+                {nameof(NpcMonsterDto.Race), chunk => Convert.ToByte(chunk["RACE"][0][2])},
+                {nameof(NpcMonsterDto.RaceType), chunk => Convert.ToByte(chunk["RACE"][0][3])},
+                {nameof(NpcMonsterDto.Element), chunk => Convert.ToByte(chunk["ATTRIB"][0][2])},
+                {nameof(NpcMonsterDto.ElementRate), chunk => Convert.ToInt16(chunk["ATTRIB"][0][3])},
+                {nameof(NpcMonsterDto.FireResistance), chunk => Convert.ToSByte(chunk["ATTRIB"][0][4])},
+                {nameof(NpcMonsterDto.WaterResistance), chunk => Convert.ToSByte(chunk["ATTRIB"][0][5])},
+                {nameof(NpcMonsterDto.LightResistance), chunk => Convert.ToSByte(chunk["ATTRIB"][0][6])},
+                {nameof(NpcMonsterDto.DarkResistance), chunk => Convert.ToSByte(chunk["ATTRIB"][0][7])},
+                {nameof(NpcMonsterDto.MaxHp), chunk => Convert.ToInt32(chunk["HP/MP"][0][2]) + _basicHp[Level(chunk)]},
+                {nameof(NpcMonsterDto.MaxMp), chunk => Convert.ToInt32(chunk["HP/MP"][0][3]) + Convert.ToByte(chunk["RACE"][0][2]) == 0 ? _basicPrimaryMp[Level(chunk)] : _basicSecondaryMp[Level(chunk)]},
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Split this 203 characters long line (which is greater than 200 authorized).](https://app.codacy.com/app/NosCoreIO/NosCore/pullRequest?prid=4512441)